package.json

```diff
@@ -6,8 +6,10 @@
   { "type"      : "git"
   , "url"       : "git://github.com/tim-smart/node-closure.git"
   }
+, "dependencies": {
+    "google-closure-compiler": "x"
```

Comment:
Before the [jar was included](https://github.com/tim-smart/node-closure/commit/e8d5fc8393361751619eb9cbf68e9f27284d66d4) so versions were locked to whatever was included. Now that it's an external dep, `20150901.0.0`, you can still lock to a version.

It should also be noted that despite the name, [google-closure-compiler](https://www.npmjs.com/package/google-closure-compiler), is not maintained or managed by Google but rather some rando dude.